samples/apps/copilot-chat-app/webapi/Skills/ChatSkill.cs

```diff
@@ -421,15 +423,18 @@ private string OptimizeOpenApiSkillJson(string jsonContent, int tokenLimit, Plan
         jsonContent = Regex.Replace(jsonContent.Trim(), @"[\n\r]", string.Empty);
         var document = JsonDocument.Parse(jsonContent);
         string lastSkillInvoked = plan.Steps[^1].SkillName;
+        string lastSkillOperationInvoked = plan.Steps[^1].Name;
```

Comment:
nit: to keep SK terminology consistent since Operations are parsed into Functions
```suggestion
        string lastSkillFunctionInvoked = plan.Steps[^1].Name;
```